lib/statsd/instrument/environment.rb

```diff
@@ -123,7 +127,7 @@ def prometheus_subsystem
       end
 
       def prometheus_percentiles
-        env.fetch("STATSD_PROMETHEUS_PERCENTILES", "95,99").split(",").map(&:to_i)
+        env.fetch("STATSD_PROMETHEUS_PERCENTILES", "").split(",").map(&:to_i)
```

Comment:
Not useful to us as we prefer histograms (because pXX values from a single process aren't really relevant)